src/webview/components/Atoms/Button/Button.tsx

```diff
@@ -7,7 +7,7 @@ type ButtonProps = {
   variant?: 'filled' | 'outlined';
 };
 
-const Button: React.FC<ButtonProps> = ({ text, onClick, variant = 'filled' }) => {
```

Comment:
Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭